iocage_lib/ioc_start.py

```diff
@@ -367,6 +367,9 @@ def __start_jail__(self):
             x for x in net
             + [x for x in parameters if '1' in x]
             + [
+                _sysvmsg,
+                _sysvsem,
+                _sysvshm,
```

Comment:
Good catch, can we follow the other formatting and move these under the name?

Reply:
Sorry, I do not get what you mean.
Can you please suggest a specific code change.

Reply:
Sure remove them from here and look at my suggestion